mindocr/postprocess/det_fce_postprocess.py

```diff
@@ -270,12 +269,14 @@ class FCEPostprocess(DetBasePostprocess):
         alpha (float): Weight parameter for reconstruction loss. Default is 1.0.
         beta (float): Weight parameter for stability loss. Default is 1.0.
         box_type (str): Text representation type. Default is "poly".
+        rescale_fields: name of fields to scale back to the shape of the original image.
```

Comment:
Sort API docs to match the actual input order of the `FCEPostprocess`.

Reply:
done.